packages/client/hmi-client/src/services/jupyter.ts

```diff
@@ -191,3 +194,109 @@ export interface IMessageHistory {
 	message: string;
 	messageType: string;
 }
+
+class KernelMessage extends EventEmitter {
+	// Make fluent interface
```

Comment:
I thought this is a well known coding/engineering term ... since we already use it in server (via lombok) and on the UI (via D3). I will add a link to the comment. 


https://en.wikipedia.org/wiki/Fluent_interface